docs/reference/supported-environments.md

```diff
@@ -89,7 +89,7 @@ Requirements for the components can be seen below:
 | Operate     | OpenJDK 17+  | Elasticsearch 8.9+<br/>Amazon OpenSearch 2.5.x                                                                                                                                                                    |
 | Tasklist    | OpenJDK 17+  | Elasticsearch 8.9+<br/>Amazon OpenSearch 2.5.x                                                                                                                                                                    |
 | Identity    | OpenJDK 17+  | Keycloak 22.x, 23.x<br/>PostgreSQL 14.x, 15.x or Amazon Aurora PostgreSQL 13.x, 14.x, 15.x (required for [certain features](/self-managed/identity/deployment/configuration-variables.md#database-configuration)) |
-| Optimize    | OpenJDK 17+  | Elasticsearch 8.9+                                                                                                                                                                                                |
+| Optimize    | OpenJDK 17+  | Elasticsearch 8.9+<br/>Amazon OpenSearch 2.5.x                                                                                                                                                                    |
```

Comment:
Just wondering, what is the reason for 2.5 as minimum here? Are all 2.x versions still in support?

Reply:
That's the mandate we got for all tools (Operate, Optimize and Tasklist). In theory you are correct, according to the OpenSearch guideline (https://opensearch.org/releases.html), `By default, versions will remain under maintenance until the next major version enters maintenance, or 1 year passes, whichever is longer.`, which would mean that 2.5.0 and 2.6.0 are no longer supported. But since we have successfully tested with those versions and the other tools have also left the doc unchanged, I don't see much value in Optimize being a special case in this page. Therefore I would leave it as is

Reply:
Not sure I interpret the docs in the same way, but also happy for Optimize to just be consistent here rather than extensive. No change needed, was more of an open question 🙂